caravel/assets/javascripts/SqlLab/index.jsx

```diff
@@ -27,9 +28,20 @@ if (process.env.NODE_ENV === 'dev') {
 let store = createStore(sqlLabReducer, initialState, enhancer);
 
 // jquery hack to highlight the navbar menu
-$('a[href="/caravel/sqllab"]').parent().addClass('active');
+// $('a[href="/caravel/sqllab"]').parent().addClass('active');
```

Comment:
This always highlights the 'SQL Lab/My Queries' in the dropdown menu when 'Query Search' is selected. Not sure how to work around this